infrap4d/CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ endif()
 
 set_install_rpath(infrap4d $ORIGIN/../lib ${SDE_ELEMENT} ${DEP_ELEMENT})
 
+target_sources(infrap4d PRIVATE $<TARGET_OBJECTS:daemon_o>)
```

Comment:
It wasn't omitted. There was a merge conflict with
```
set_install_rpath(infrap4d $ORIGIN/../lib ${SDE_ELEMENT} ${DEP_ELEMENT})
```
which I resolved in favor of `set_install_path`

Reply:
Wait a minute ... did I delete the wrong line?

Reply:
Ohh okay. I think so. I don't see target_sources line in latest code